cmd/operator/main.go

```diff
@@ -203,7 +203,7 @@ func Main() int {
 
 	wg, ctx := errgroup.WithContext(ctx)
 
-	wg.Go(func() error { return o.Run(ctx.Done()) })
+	wg.Go(func() error { return o.Run(ctx, ctx.Done()) })
```

Comment:
I would rather change the signature of Run() since it could use `ctx.Done()`

```suggestion
	wg.Go(func() error { return o.Run(ctx) })
```

Reply:
Is `operator.Run` trying to satisfy an interface that has `Run(ctx, stopCh <-struct{})`?